frontend/tests/playwright.config.ts

```diff
@@ -0,0 +1,27 @@
+import { defineConfig, devices } from '@playwright/test';
+import type { ConfigOptions } from '@nuxt/test-utils/playwright';
+import dotenv from 'dotenv';
+import path from 'path';
+import { fileURLToPath } from 'url';
+
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = path.dirname(__filename);
+
+dotenv.config({ path: path.resolve(__dirname, './tests/.env') });
+
+export default defineConfig<ConfigOptions>({
+  outputDir: './results',
+  testMatch: '**/*.spec.ts',
+  timeout: 30000,
+  use: {
+    nuxt: {
+      // host: 'http://localhost:3000',
+    },
+  },
```

Comment:
we have to make the `url` configurable from the depending on the `deploytarget`/`environment`...we could use the [baseUrl](https://playwright.dev/docs/test-configuration#basic-configuration) or we could use `host`....but the most important thing is that we make it `configurable`:

```suggestion
  use: {
    baseURL: process.env.NUXT_PUBLIC_DOMAIN,
    nuxt: {
      // host: 'http://localhost:3000',
    },
  },
```
there are several possibilities to make it work that way, reach out to me, as we should not leak `staging urls here`